comment_spell_check/utils/create_checker.py

```diff
@@ -0,0 +1,76 @@
+"""Create a case sensitive spell checker with the English dictionary and
+additional dictionaries if provided.
+"""
+
+import sys
+import logging
+import importlib.resources
+import spellchecker
+
+
+def create_checker(dict_list: list[str] = None, logger: logging.Logger = None):
+    """Create a case sensitive spell checker with the English dictionary and
+    additional dictionaries if provided."""
+
+    # create an empty SpellChecker object, because we want a case
+    # sensitive checker
+    checker = spellchecker.SpellChecker(language=None, case_sensitive=True)
+
+    # load the English dictionary
+    lib_path = importlib.resources.files(spellchecker)
+    english_dict = str(lib_path) + "/resources/en.json.gz"
+    logger.info(f"Loading English dictionary from: {english_dict}")
+    checker.word_frequency.load_dictionary(english_dict)
+
+    # load the additional dictionaries
+    if not isinstance(dict_list, list):
+        return checker
+    if len(dict_list) > 0:
+        for d in dict_list:
+            logger.info(f"Loading additional dictionary from: {d}")
+            checker.word_frequency.load_text_file(d)
+
+    return checker
+
+
+if __name__ == "__main__":
+
+    # set up logging
+    lg = logging.getLogger(__name__)
```

Comment:
The common convention is to create a logger per module/file like

`logger = logging.getLogger(__name__)`

This should create a hierarchy of loggers where the verbosity can be changes for the entire package or on a module level.